### DISCUSSION TITLE
New icon

### DISCUSSION THREAD

**Discussion by 2br-2b** [Franciscan-CS-Club/fix-franciscan-ordering#6] [General]
We need a better icon than the current one:

<img src="https://raw.githubusercontent.com/Franciscan-CS-Club/fix-franciscan-ordering/main/icons/icon.svg" height=300>

I grabbed this from a public domain site (https://publicdomainvectors.org/en/free-clipart/No-fast-food-vector-symbol/74506.html), but I'm sure we can do better.

We should make sure that:
1) We communicate what the extension does;
2) We don't have a copyright violation

**Comment by AaronBangs** [comment]
![burger](https://user-images.githubusercontent.com/58962483/110483153-711c9080-80b7-11eb-9738-8cf6041a13e4.png)
